### PR TITLE
[ADD] l10n_tw

### PR DESCRIPTION
**Pull request by tong-odoo**
- add Taiwan localisation

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_tw/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "data/account_chart_template_data.xml",
+        "data/account.tax.group.csv",
+        "data/account_tax_template_data.xml",
+        "data/account_chart_template_configure_data.xml",
+        "data/res_currency_data.xml",
+        "data/res_country_data.xml",
+        "data/res.city.csv",
+    ],
+    'post_init_hook': 'load_translations',
+    'license': 'LGPL-3',
```
Comment (fja-odoo) [inline addons/l10n_tw/__manifest__.py:29]: 2 license ?

Comment (tong-odoo) [reply addons/l10n_tw/__manifest__.py:28]: Updated

File: addons/l10n_tw/data/account.account.template.csv [hunk 2]
```diff
@@ -0,0 +1,15 @@
+id,chart_template_id/id,code,name,user_type_id/id,reconcile
```
Comment (FlorianGilbert) [inline addons/l10n_tw/data/account.account.template.csv:1]: Why are there so few accounts?

File: addons/l10n_tw/data/account_tax_template_data.xml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                    'account_id': ref('tw_126800'),
+                }),
+            ]"/>
+    </record>
+    <record id="tw_tax_purchase_inc_5" model="account.tax.template">
+        <field name="chart_template_id" ref="l10n_tw_chart_template" />
+        <field name="name">GST Inc Purchase (5%)</field>
+        <field name="description">GST Inclusive Purchases</field>
+        <field name="type_tax_use">purchase</field>
+        <field name="tax_scope">consu</field>
```
Comment (FlorianGilbert) [inline addons/l10n_tw/data/account_tax_template_data.xml:108]: Just asking why there is a tax_scope here and not for GST Inc Sale (5%) ? (Maybe stupid question)

**Review by FlorianGilbert — CHANGES_REQUESTED** [review]
Hello

Could you please add the task id and explain what you do in your commit according to [guidelines](https://www.odoo.com/documentation/master/developer/misc/other/guidelines.html#commit-message-structure).

Thank you :)

File: addons/l10n_tw/data/account.account.template.csv (hunk 2 above)
Comment (tong-odoo) [reply addons/l10n_tw/data/account.account.template.csv:1]: From the TW community they have another file of account.account.template for other usage, but this file is the most general one. I can ask the FC to check again if there is any additional account.account need to add inside.

File: addons/l10n_tw/data/account_tax_template_data.xml (hunk 3 above)
Comment (tong-odoo) [reply addons/l10n_tw/data/account_tax_template_data.xml:108]: Oh maybe i mix up with other localisation, this shouldn't be here.

**Comment by tong-odoo** [comment]
@FlorianGilbert Updated all of the issues, please feel free to let me know if there is anymore~ Thanks~ :)

**Review by FlorianGilbert — APPROVED** [review]
Look good to me
Thank you.

@robodoo r+

**Pull request closed without merging**